draft-ietf-rats-msg-wrap.md

```diff
@@ -249,13 +249,23 @@ The `tag-cm-cbor` macro takes the CDDL definition of the associated conceptual m
 {: #fig-cddl-cbor-tag artwork-align="left"
    title="CDDL definition of the Tag CMW macros"}
 
-To add a new CMW, the `$cbor-tag` type socket is extended with a new instance of the Tag CMW macro.
-For example, to associate conceptual messages of type `my-evidence` with CBOR tag `1668576819`, one would extend `$cbor-tag` as follows:
+### How To Plug in a New Tag CMW
+
+To plug a new Tag CMW into the CDDL defined in {{collected-cddl}}, the `$cbor-tag` type socket must be extended with a new instance of the Tag CMW macro.
```

Comment:
maybe ref or elaboration to CMW macro

Reply:
The macro is defined 10 lines above, in the parent section.  I reckon the reader's stack should be correctly configured to understand what we are talking about :-)